keybot/keybot.go

```diff
@@ -67,8 +74,15 @@ func (k *keybot) Run(bot slackbot.Bot, channel string, args []string) (string, e
 	env := launchd.NewEnv(home, path)
 	switch cmd {
 	case cancel.FullCommand():
+		if *cancelWin {
+			jenkins.StopBuild(*cancelLabel)
+			out := "Issued cancel"
+			if *cancelLabel != "" {
```

Comment:
Is cancelLabel required? If nothing happens when you don't specify, return an error?
```go
if *cancelLabel == "" {
  return nil, fmt.Errorf("Need to specify job name (label)")
}
```